rules/integrations/kubernetes/execution_pod_created_in_kube_namespace.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2022/05/23"
+maturity = "production"
+updated_date = "2022/05/23"
+integration = "kubernetes"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects when a pod is created in the kube-system or kube-public namespace. These default namespaces are reserved for cluster services and cannot be deleted. Only system administrators should edit resources here and as a best practice user resources should not be deployed in either 'kube' namespace. 
+
+The kube-system namespace is reserved for objects created by the Kubernetes system. Adversaries can use this namespace to deploy malicious pods or containers with names similar to those of default system components in order to evade detection. 
+
+The kube-public namespace is intended for public resources and unauthenticated user access. A user pod created here should be further investigated to determine its validity in the cluster. 
+
+Adversaries can deploy malicious pods in these namespaces to maintain persistence, evade detection, or facilitate execution in a cluster. 
+"""
+false_positives = [
+    """
+    There are legitimate uses of these namespaces that could trigger alerts. The Kubernetes systems deploys objects to kube-system namespace and administrators may deploy objects to kube-public that are intended for public access.
+    """,
+]
+index = ["filebeat-*", "logs-kubernetes.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Pod Created in Kube Namespace"
+note = """## Config
+
+The Kubernetes Fleet integration with Audit Logs enabled, Filebeat module, or similarly structured data is required to be compatible with this rule."""
```

```suggestion
The Kubernetes Fleet integration with Audit Logs enabled or similarly structured data is required to be compatible with this rule."""
```

This _may_ require modification to the unit test